v/vault/build_info.json

```diff
@@ -0,0 +1,16 @@
+{
+    "package_name" : "vault",
+	"github_url":"https://github.com/hashicorp/vault.git",
+    "package_dir": "v/vault/Dockerfiles/",
+    "docker_cmd" :"docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
+    "build_script" : "vault_ubi_8.5.sh",
+    
+    "v1.1.3" : {
+        "dir":"1.1.3-ubi8"
```

Comment:
As we are using VAULT_VERSION as ARG in Dockerfile, please add below snippet to the each version:

"args": {
            "VAULT_VERSION":"$PACKAGE_VERSION"
        }
 
Please refer https://github.com/ppc64le/build-scripts/blob/master/k/kafka/build_info.json#L17 .

Reply:
Made changes accordingly